src/main/java/net/rptools/maptool/client/AppActions.java

```diff
@@ -498,6 +500,12 @@ protected void executeAction() {
           }
 
           MapTool.serverCommand().enforceZone(renderer.getZone().getId());
+          List<Token> libTokens = EventMacroUtil.getEventMacroTokens(ON_CHANGE_MAP_CALLBACK);
```

Comment:
I don't think this would be needed if you are going to use the onZoneLoaded event

Reply:
In my testing, when I did not have that code in AppActions, the event wouldn't trigger on a Tools -> Enforce Players to Current Map.  So a GM can change maps and the event fires.  Then if a GM pulls all the players to the map their on, the event fires for the players but without this, it wouldn't fire _again_ for the GM (as they're on the map already)

Reply:
This has been removed (2b is after spotless).